src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/Charts/MultiScatterChart/MultiScatterChart.utils.ts

```diff
@@ -52,6 +54,8 @@ export const getOptions = ({
     },
     scales: {
       y: {
+        type: (useCategory ? 'category' : undefined) as 'category' | undefined,
```

Comment:
Mayme for this case you need to introduce the enum from library interface and use its specific value `ScalesType.category`:
```
enum const ScalesType = keyof CartesianScaleTypeRegistry;
```